test/validation/api/init/init_main.c

```diff
@@ -57,12 +57,14 @@ static void init_test_defaults(void)
 
 	odp_init_param_init(&param);
 
+	memset(&instance, 0, sizeof(instance));
 	ret = odp_init_global(&instance, &param, NULL);
```

Comment:
The contents of `odp_instance_t` should be initialized by the `odp_init_global()` implementation.

Reply:
But maybe this code should not fully trust it since this is API validation test. The commit comment should clarify the purpose of this change since now we can only guess.

The memsets would help if odp_instance_t was actually a structure with internal padding because of the memcmp, but maybe that is a bit far-fetched.

Reply:
Yeah, I was assuming this fix was done to fix LTO build, but that was just a guess since the commit message is insufficient.